util/ArrayUtil.js

```diff
@@ -0,0 +1,7 @@
+module.exports = class ArrayUtil {
```

Comment:
if its included in the `util` directory, the name `ArrayUtil`'s inclusion of `Util` is redundant.  I think `util/array` is probably not the name we want to give this file, but I'm not sure which direction to go.  Below, the `GetIntRange` isn't really a number utility either, its just a helper function.  Maybe they belong in util/index

Object and String utils seem appropriate.

Reply:
I think it's an instance of acceptable redundancy because you don't want to name a class after a js type, yet I can't think of anything else that appropriately describes it. So `GetIntRange` in util/index? What would that file be? Just an export of a single generic helper class called `Util`?